addOns/spider/src/main/java/org/zaproxy/addon/spider/filters/DefaultParseFilter.java

```diff
@@ -55,6 +55,10 @@ public class DefaultParseFilter extends ParseFilter {
     private static final Pattern SITEMAP_FILENAME_PATTERN =
             Pattern.compile(".*/sitemap.xml$", Pattern.CASE_INSENSITIVE);
 
+    /** a pattern to match the .DS_Store files. */
+    private static final Pattern DS_STORE_FILENAME_PATTERN =
+            Pattern.compile(".*/.DS_Store$", Pattern.CASE_INSENSITIVE);
```

Comment:
Do I need to escape here?
```suggestion
            Pattern.compile(".*/\.DS_Store$", Pattern.CASE_INSENSITIVE);
```

Reply:
It should otherwise it matches any char, but needs to be `\\`.